src/parameterizations/lateral/MOM_Zanna_Bolton.F90

```diff
@@ -712,24 +883,22 @@ subroutine compute_stress_divergence(u, v, h, diffu, diffv, dx2h, dy2h, dx2q, dy
       enddo ; enddo
     endif
 
-    ! Evaluate 1/h x.Div(h S) (Line 1495 of MOM_hor_visc.F90)
-    ! Minus occurs because in original file (du/dt) = - div(S),
-    ! but here is the discretization of div(S)
+    ! Evaluate du/dt=1/h x.Div(h T) (Line 1495 of MOM_hor_visc.F90)
     do j=js,je ; do I=Isq,Ieq
       h_u = 0.5 * (G%mask2dT(i,j)*h(i,j,k) + G%mask2dT(i+1,j)*h(i+1,j,k)) + h_neglect
-      fx = -((G%IdyCu(I,j)*(Mxx(i,j) - Mxx(i+1,j)) + &
-              G%IdxCu(I,j)*(dx2q(I,J-1)*Mxy(I,J-1) - dx2q(I,J)*Mxy(I,J))) * &
+      fx =  ((G%IdyCu(I,j)*(Mxx(i+1,j) - Mxx(i,j)) + &
+              G%IdxCu(I,j)*(dx2q(I,J)*Mxy(I,J) - dx2q(I,J-1)*Mxy(I,J-1))) * &
```

Comment:
We will eventually need to add parentheses to lines 890 and 901 to give rotational symmetry with fused-multiply-adds are enabled, so that line 890 would read `G%IdxCu(I,j)*((dx2q(I,J)*Mxy(I,J)) - (dx2q(I,J-1)*Mxy(I,J-1)))) * &`